dask_ml/model_selection/_incremental.py

```diff
@@ -635,7 +641,10 @@ async def _fit(self, X, y, **fit_params):
             context = dummy_context()
 
         X, y, scorer = self._validate_parameters(X, y)
-        X_train, X_test, y_train, y_test = self._get_train_test_split(X, y)
+
+        X_train, X_test, y_train, y_test = self._get_train_test_split(
+            X, y, shuffle=True
+        )
```

Comment:
```suggestion

        X_train, X_test, y_train, y_test = self._get_train_test_split()
```

Reply:
`_get_train_test_split` wasn't using `shuffle`. It's OK to just use the hard-coded value.